extensions-contrib/kafka-segment-metadata-emitter/src/main/proto/DruidSegmentEvent.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+package streamingaggregates.proto;
```

Comment:
shall we change the package name to druid? Not sure what's the standard we are using for proto packages